blockchain/beacon/src/mock_beacon.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::{Beacon, BeaconEntry};
+use async_trait::async_trait;
+use byteorder::{BigEndian, WriteBytesExt};
+use encoding::blake2b_256;
+use std::error::Error;
+use std::time::Duration;
+
+pub struct MockBeacon {
+    interval: Duration,
+}
+
+impl MockBeacon {
+    pub fn new(interval: Duration) -> Self {
+        MockBeacon { interval }
+    }
+    fn entry_for_index(index: u64) -> BeaconEntry {
+        let mut buf: Vec<u8> = Vec::with_capacity(8);
+        buf.write_u64::<BigEndian>(index).unwrap();
```

Comment:
```suggestion
        let mut buf = [0; 8];
        BigEndian::write_u64(&mut buf, index);
```
Saves an allocation. You will need to `use byteorder::ByteOrder` as well